.travis.yml

```diff
@@ -28,8 +28,8 @@ jobs:
       name: "Type Checking"
       script: mypy lib/wit/*.py
     - stage: test
-      name: "Regression Tests"
```

Comment:
Just for the record, since I'm the main one that has been asking for the terminology change: I think the phrase "[regression test](https://en.wikipedia.org/wiki/Regression_testing)" is fine, since it is refers to a test that checks that there are no regressions in behavior.

What I am against is using the word "regression" to refer to the test itself, since it means the exact opposite of what we want. To regress means to return to a previous state, not to run tests that hopefully prevent returning to a previous state.

Reply:
Hahah yeah fair enough, I'll restore it to "Regression Tests" but keep everything else the same.